internal/util/fetch/fetch_test.go

```diff
@@ -433,6 +426,69 @@ func TestCommand_Run_tag(t *testing.T) {
 	})
 }
 
+func TestCommand_Run_subdir_at_tag(t *testing.T) {
```

Comment:
We should test additional cases here. Examples are:
 * Verify that the most specific version is chosen if there are multiple possible matches.
 * What happens if there are no match?
 * What happens if the provided `dir` doesn't have the leading `/`?

Reply:
Added tests.